drivers/SmartThings/zigbee-motion-sensor/src/sengled/init.lua

```diff
@@ -0,0 +1,56 @@
+local clusters = require "st.zigbee.zcl.clusters"
+local battery_defaults = require "st.zigbee.defaults.battery_defaults"
+
+local IASZone = clusters.IASZone
+local PowerConfiguration = clusters.PowerConfiguration
+
+local FINGERPRINTS = {
+  { mfr = "sengled", model = "E1M-G7H" }
+}
+
+local CONFIGURATIONS = {
+  {
+    cluster = IASZone.ID,
+    attribute = IASZone.attributes.ZoneStatus.ID,
+    minimum_interval = 30,
+    maximum_interval = 3600,
+    data_type = IASZone.attributes.ZoneStatus.base_type,
+    reportable_change = 1
+  },
+  {
+    cluster = PowerConfiguration.ID,
+    attribute = PowerConfiguration.attributes.BatteryVoltage.ID,
+    minimum_interval = 30,
+    maximum_interval = 3600,
+    data_type = PowerConfiguration.attributes.BatteryVoltage.base_type,
+    reportable_change = 1
+  }
+}
+
+local is_sengled_products = function(opts, driver, device, ...)
+  for _, fingerprint in ipairs(FINGERPRINTS) do
+    if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
+      return true
+    end
+  end
+  return false
+end
+
+local function device_init(driver, device)
+  battery_defaults.build_linear_voltage_init(2.2, 3.0)(driver, device)
+
+  for _, attribute in ipairs(CONFIGURATIONS) do
```

Comment:
@greens should a test_sengled_sensor.lua file be added to test that these configurations are set correctly? It looks like that is the unique behavior in this subdriver

Reply:
that would be nice to see